src/ext/default.c

```diff
@@ -140,8 +140,8 @@ static double TFIDFNormDocLenScorer(const ScoringFunctionArgs *ctx, const RSInde
 /******************************************************************************************
  *
  * BM25 Scoring Functions
- *
- * https://en.wikipedia.org/wiki/Okapi_BM25
+ * NOTE: this is a legacy *non-standard* computation of BM25, and is deprecated after introducing
```

Comment:
Need to mark it as deprecated [here](https://github.com/RediSearch/RediSearch/blob/master/docs/docs/advanced-concepts/scoring.md#bm25)
And also list the new `BM25STD`